CONFIG.md

```diff
@@ -41,3 +41,5 @@ This list will be updated as properties are migrated:
 | Old property name | New property name | Notes |
 | -------- | ------- | ------- |
 | org.jitsi.videobridge.health.INTERVAL | videobridge.health.interval | The new config models this as a duration, rather than an amount of milliseconds |
+| org.jitsi.videobridge.health.TIMEOUT | videobridge.health.timeout | The new config models this as a duration, rather than an amount of milliseconds |
+| org.jitsi.videobridge.health.STICKY_FAILURES | videobridge.health.sticky-failures | |
```

Comment:
Do we want to have something in this readme that says what the valid values are? I assume this is a boolean?

Reply:
the intention here was just to document migration-related data (so just the mappings of old to new and any applicable notes), not a full documentation of all the properties

Reply:
Ok.

I wonder if we _should_ have full documentation of config properties (so in particular we can distinguish between ones which are documented and which ones we consider internal) but that's a separate issue.

Reply:
yeah i agree, i have been trying to make the defaults file as self-documented as possible (with comments) in hopes we can avoid the need for a separate doc, so maybe a naming convention to separate internal from external?

Reply:
Sounds reasonable to me.  (Let's decide on a naming convention when we get to something we think should be internal.)